src/styles/blocks.css

```diff
@@ -23,3 +23,13 @@
     }
   }
 }
+
+.simple-box {
+  position: relative;
+  margin: 0 0 1rem 0;
+  padding: 1rem;
+  border: 1px solid rgba(10, 10, 10, 0.25);
+  border-radius: 0;
+  background-color: var(--colors-nearly-white);
+  color: #0a0a0a;
+}
```

Comment:
Couldn't be done with an `Callout` using `alt` and no type/icon?

Reply:
Yeah, it's possible, but I think was more of a semantic decision. as the components using this style won't necessarily be sending a message, and just need the style, so technically is not a `Callout`